src/aliceVision/version.hpp

```diff
@@ -18,3 +18,23 @@
 #define ALICEVISION_VERSION_STRING ALICEVISION_TO_STRING(ALICEVISION_VERSION_MAJOR) "." \
                              ALICEVISION_TO_STRING(ALICEVISION_VERSION_MINOR) "." \
                              ALICEVISION_TO_STRING(ALICEVISION_VERSION_REVISION)
+
+                             
+
+inline bool isVersionOlder(const std::vector<uint32_t> & reference, const std::vector<uint32_t> & compare)
```

Comment:
seems unused:
version is a string, a define or a Vec3, but I do not see the case with std::vector.